scripts/prepare_weights.sh

```diff
@@ -1,12 +1,19 @@
 #!/bin/bash
+# Source the download script which is likely responsible for downloading files from URLs
 source scripts/download.sh
 
 # Prepare dirs
+# Set the base directory for the models
 MODEL_DIR=models
+# Set the directory for the llama model weights
 LLAMA_DIR=$MODEL_DIR/llama
+# Set the directory for the vicuna delta weights
 VICUNA_DIR=$MODEL_DIR/vicuna-7b-delta-v0
+# Set the directory for the MiniGPT-4 model weights
 MINIGPT4_DIR=$MODEL_DIR/minigpt4
+# Set the directory for the output model weights
 OUTPUT_DIR=$MODEL_DIR/output
+# Create the directories if they do not already exist
 mkdir -p $LLAMA_DIR
 mkdir -p $OUTPUT_DIR
```

Comment:
Can we add a if clause to verify if the directory already exists or not, and only mkdir if it doesn't exists?

Reply:
Sweep: Can we add a if clause to verify if the directory already exists or not, and only mkdir if it doesn't exists?